article/index.md

```diff
@@ -2,19 +2,22 @@
 
-For our internal infrastructure, it is no surprise that we also use Pulumi for this. At this early phase of our company (we founded 1 January!), it is hard to foresee how we will evolve. With Pulumi, we write code in a regular programming language. A good habit when programming is to prevent [premature optimization](http://wiki.c2.com/?PrematureOptimization) as you can't predict the future. Another good habbit when writing code is [red-green-refactor](https://en.wikipedia.org/wiki/Test-driven_development). You take the current code, write a test for the new requirement which is initially failing (red), you implement the code in the most straightforward way to make the test succeed (green) and you complete the cycle by refactoring the code to keep the design in a proper shape.
+At Cumundi we build reusable libraries for our customers to set up their infrastructure with a lot of best practices integrated. These best practices span the range of non-functionals, which are most of the time not taken up with the same focus as application feature development. Where these non-functionals can be integrated in infrastructure setups, we make sure these are taken care of in our code libraries. We focus on all three big cloud providers and try to provide application focused infrastructure libraries.
+
+For our internal infrastructure it is no surprise that we also use Pulumi. At this early phase of our company (we founded January 1st!), it is hard to foresee how we will evolve. With Pulumi, we write code in a regular programming language. A good habit when programming is to prevent [premature optimization](http://wiki.c2.com/?PrematureOptimization) as you can't predict the future. Another good habbit when writing code is [red-green-refactor](https://en.wikipedia.org/wiki/Test-driven_development). You take the current code, write a test for the new requirement which is initially failing (red), you implement the code in the most straightforward way to make the test succeed (green) and you complete the cycle by refactoring the code to keep the design in a proper shape.
 
 In this blog post, I will show such a cycle for Pulumi code with a reduced version of the code we use to set up the internal infrastructure we use ourselves for each customer. In this example, I will focus on one specific Pulumi resource property: [`aliases`](https://www.pulumi.com/docs/intro/concepts/programming-model/#aliases)
 
 All the code is [published here](https://github.com/cumundi/pulumi-refactoring-aliases) if you want to follow along with a full project setup. Every step described here is committed as a separate branch with the starting point on `master`, the default branch.
+@Ringo: hoe zit het met Osimis? Gaan die hun code herkennen, of omgekeerd, gaan ze zich afvragen waarom je voor Cumundi een andere versie maakt dan voor hen (die misschien beter is)?
```

Comment:
@majadhondt I deliberately took code we only use internally at Cumundi. No overlap here.